LAYOUT_STYLE_GUIDE.md

```diff
@@ -58,11 +58,17 @@ CUSTOM SECTION CONTENT.
 
 <div class="lesson-content__panel" markdown="1">
 
+#### CUSTOM ASSIGNMENT HEADING
+
 1. A RESOURCE OR EXERCISE ITEM
    - AN INSTRUCTION ITEM
 
 </div>
 
+### POST-ASSIGNMENT SECTION HEADING
```

Comment:
Yeah this was something else that was landed on I think after conversations at the time. Believe the intent is for the stuff that needs to be customized by the user to stand out more so that it doesn't as easily get forgotten and left in